frontend/src/components/jobs/result/utils/SaveAsPlaybooksForm.jsx

```diff
@@ -60,20 +60,22 @@ export function SaveAsPlaybookForm({ onFormSubmit }) {
           <FormGroup row className="d-flex flex-wrap">
             <Col>
               <div className="p-3">
-                <Label className="required" for="forminput-username" md={12}>
+                <Label className="required" for="forminput-name" md={12}>
                   Playbook name
                 </Label>
                 <Input
                   autoFocus
                   id="forminput-name"
                   type="text"
                   name="name"
+                  onBlur={formik.handleBlur}
                   onChange={formik.handleChange}
                 />
+                {formik.touched.name && <small>{formik.errors.name}</small>}
```

Comment:
I suggest to write this test message in red, also the description field is required